app/shared/containers/Tools.js

```diff
@@ -56,16 +57,19 @@ import * as WalletsActions from '../actions/wallets';
 
 const paneMapping = [
   {
+    alwaysAvailable: true,
```

Comment:
I think the way to go about this is to have a field called:

`requiresFeature` (or something along those lines). 

The absence of the field indicates it's always available, and if it has a `requiresFeature` flag, it ensures that feature exists before enabling it. 

This probably doesn't matter a ton though because the entire tools page will likely go away with Anchor - but food for thought :)